internal/controller/statusreport/statusreport_adapter.go

```diff
@@ -184,6 +186,50 @@ func (a *Adapter) EnsureSnapshotFinishedAllTests() (controller.OperationResult,
 	return controller.ContinueProcessing()
 }
 
+// EnsureGroupSnapshotCreationStatusReportedToGitProvider is an operation that will ensure the the group snapshot creation status is report to component snapshot
+func (a *Adapter) EnsureGroupSnapshotCreationStatusReportedToGitProvider() (controller.OperationResult, error) {
+	// Only report status for component Snapshots
```

Comment:
Should this comment state following?
// Only report status for group Snapshots
or something like: skip in case of component snapshot?

Reply:
Fixed. Thanks.